doc/source/commands/flight/systems.rst

```diff
@@ -34,7 +36,16 @@ RCS and SAS
 
     Turns the SAS **on** or **off**, like using ``T`` at the keybaord::
 
-        SAS ON.
+        SAS ON. // same as SET SAS TO TRUE.
+        SAS OFF. // same as SET SAS TO FALSE.
+        PRINT SAS.  // prints either "True" or "False".
```

Comment:
Technically not part of the remit of the issue, but I didn't like how terse the example was originally, and wanted to make clear the point that SAS is really just a boolean variable and can be used like one.